drivers/hmis/spec/support/ce_system_test_helper.rb

```diff
@@ -0,0 +1,124 @@
+###
+# Copyright 2016 - 2025 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+# Shared context for CE system tests.
+# Sets up a correctly configured target project and admin/provider users with example permissions.
+RSpec.shared_context 'ce system test helper' do
+  # Use find_or_create_by since some tests create the data source in a before(:all) block for performance reasons.
+  let!(:ds1) { GrdaWarehouse::DataSource.find_or_create_by!(hmis: 'localhost', source_type: :sftp, name: 'HMIS', short_name: 'HMIS') }
```

Comment:
I don't know why the tests use `sftp` as the source type, since OP HMIS is not an sftp source. Since its not relevant here, maybe we can change this to:

```suggestion
  let!(:ds1) { GrdaWarehouse::DataSource.find_or_create_by!(hmis: 'localhost', name: 'HMIS') }
```
?